spring-cloud-gcp-samples/spring-cloud-gcp-config-sample/src/main/resources/bootstrap.properties

```diff
@@ -11,7 +8,7 @@ spring.cloud.gcp.config.name=myapp
 spring.cloud.gcp.config.profile=prod
 
 # Config server API timeout in milliseconds, default : 60000 (1 minute)
```

Comment:
This comment here says 60 seconds, but the setting below is 120 seconds it seems

Reply:
Right, the point is to override the default config with the setting here.